src/js/structs/PodInstance.js

```diff
@@ -0,0 +1,85 @@
+import Item from './Item';
+import PodContainer from './PodContainer';
+import PodInstanceStatus from '../constants/PodInstanceStatus';
+import StringUtil from '../utils/StringUtil';
+
+module.exports = class PodInstance extends Item {
+
+  getContainers() {
+    let containers = this.get('containers') || [];
+    return containers.map((container) => {
+      return new PodContainer(container);
+    });
+  }
+
+  getInstanceStatus() {
+    switch (this.get('status')) {
+      case 'PENDING':
+        return PodInstanceStatus.STAGED;
+
+      case 'STAGING':
+        return PodInstanceStatus.STAGED;
+
+      case 'STABLE':
+        if (this.hasHealthChecks()) {
+          if (this.isHealthy()) {
+            return PodInstanceStatus.HEALTHY;
+          } else {
+            return PodInstanceStatus.UNHEALTHY;
+          }
+        } else {
+          return PodInstanceStatus.RUNNING;
+        }
+
+      case 'DEGRADED':
+        return PodInstanceStatus.UNHEALTHY;
+
+      case 'TERMINAL':
+        return PodInstanceStatus.KILLED;
+
+      default:
+        return Object.assign(Object.create(PodInstanceStatus.NA), {
+          displayName: StringUtil.capitalize(this.get('status').toLowerCase())
+        });
+    }
+  }
+
+  isInStatus(statusArray) {
+    let myStatus = this.get('status') || 'NA';
+    return statusArray.reduce(function (matched, status) {
+      if (matched) {
+        return true;
+      }
+      return status === myStatus;
```

Comment:
Please add an empty line before returns if the `return` is not the only statement #nit